packages/cli/src/spec/mock/MockPrompt.ts

```diff
@@ -0,0 +1,59 @@
+/*
+ * Copyright 2020 Google Inc. All Rights Reserved.
+ *
+ *  Licensed under the Apache License, Version 2.0 (the "License");
+ *  you may not use this file except in compliance with the License.
+ *  You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ *  Unless required by applicable law or agreed to in writing, software
+ *  distributed under the License is distributed on an "AS IS" BASIS,
+ *  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ *  See the License for the specific language governing permissions and
+ *  limitations under the License.
+ */
+
+import {Prompt} from '../../lib/Prompt';
+
+/**
+ * A class which used for testing and which mocks user's input.
+ */
+export class MockPrompt implements Prompt {
+  async printMessage(): Promise<void> {
+    // An empty function for testing.
+  }
+
+  /**
+   * Sets the output to be the given message.
+   * @param message the message to be returned.
+   */
+  async promptInput<T>(message: string): Promise<T> {
+    return message as unknown as T;
+  }
+
+  /**
+   * Sets the output to be the given message.
+   * @param message the message to be returned.
+   */
+
+  async promptChoice<T>(message: string): Promise<T> {
```

Comment:
`promptInput`, `promptChoice` and `promptPasword` have the wrong signatures.

Reply:
It is fine, I don't get all of the inputs because I don't need them.
I can remove inputs from the end and it is fine by js..
It sends me no errors for implementing Prompt

Reply:
See my general note. I'm fine with this, but this Prompt implementation seems tighly coupled with specific tests and should be declared with them. Otherwise, we'd need to make it a general class and Peter's comment applies.